tests/bench/obex.py

```
@@ -0,0 +1,645 @@
+    exp_explicit: list[str] = [o["content"] for o in expected.get("explicit", [])]
+    exp_deductive: list[str] = [o["conclusion"] for o in expected.get("deductive", [])]
+
+    ext_explicit: list[str]
+    ext_deductive: list[str]
+    error: str | None
+    try:
+        result = await run_deriver(messages, target_peer)
+        ext_explicit = [o.content for o in result.explicit]
+        ext_deductive = [o.conclusion for o in result.deductive]
+        error = None
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)